python/src/aac/parser/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""AaC Parser and related functions submodule."""
+
+from aac.parser._parse_source import parse, get_yaml_from_source
+from aac.parser._parser_error import ParserError
+
+__all__ = (
+    ParserError.__name__,
+    parse.__name__,
+    get_yaml_from_source.__name__,
+)
```

Comment:
1. Update `parser` package to `parse`
2. Need to update parser error
  a. `aac.parser._parser_error` -> `aac.parser.parser_error`
3. Need to make the `parse_source` module public.